### PR TITLE
Migrate commits feed test

### PR DESCRIPTION
**Pull request by hennevogel**

### CODE REVIEW

**Comment by mdeniz** [comment]
Rubocop gem is outdated in Travis

File: src/api/app/controllers/webui/feeds_controller.rb [hunk 1]
```diff
@@ -15,14 +16,9 @@ def latest_updates
...
   def commits
-    @project = Project.find_by_name(params[:project])
-    if @project.nil?
-      render(file: Rails.root.join('public/404'), status: 404, layout: false, formats: [:html])
-      return
-    end
     # The sourceaccess flag is checked for the project, but not for every package
     if !User.current.is_admin? && @project.disabled_for?('sourceaccess', nil, nil)
-      render file: Rails.root.join('public/403'), formats: [:html], status: :forbidden, layout: false
+      redirect_to '/403.html', :status => :forbidden
```
Comment (hennevogel) [inline src/api/app/controllers/webui/feeds_controller.rb]: This was a problem in the controller test. It failed with some weird problem in ActionView::Resolver::Cache

File: src/api/spec/controllers/webui/feeds_controller_spec.rb [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+
+    it "assigns @project" do
+      get(:commits, { project: project, format: 'atom'  })
+      expect(assigns(:project)).to eq(project)
+    end
+
+    it "fails if project is not existent" do
+      expect do
+        get(:commits, { project: 'DoesNotExist', format: 'atom'  })
+      end.to raise_error ActiveRecord::RecordNotFound
```
Comment (bgeuken) [inline src/api/spec/controllers/webui/feeds_controller_spec.rb]: Guess we don't have a coding convention rule for this, but I would prefer {}.to here

**Comment by bgeuken** [comment]
LGTM

**Comment by mdeniz** [comment]
LGTM :+1:

**Pull request merged**